src/Orleans.Core/Configuration/Validators/ClientClusteringValidator.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+
+using Microsoft.Extensions.DependencyInjection;
+
+using Orleans.Messaging;
+using Orleans.Runtime;
+
+namespace Orleans.Configuration.Validators
+{
+    internal class ClientClusteringValidator : IConfigurationValidator
+    {
+        internal const string ClusteringNotConfigured =
+            "Clustering has not been configured. Configure clustering using one of the clustering packages, such as: "
+            + "\n  * Microsoft.Orleans.Clustering.AzureStorage for Microsoft Azure Storage"
+            + "\n  * Microsoft.Orleans.Clustering.AdoNet for ADO.NET systems such as SQL Server, MySQL, PostgreSQL, and Oracle"
+            + "\n  * Microsoft.Orleans.Clustering.DynamoDB"
+            + "\n  * Microsoft.Orleans.Clustering.ServiceFabric"
```

Comment:
ZooKeeper and Consul too?

Reply:
I can add them, I thought the list might get too long

Reply:
Never mind then.

Reply:
Well, there's no real reason to be compact here. I'll update it

Reply:
It's more important that users see something they can use